### PR TITLE
DHFPROD-9555: Unmerge via Explore Graph View

### PR DESCRIPTION
**Pull request by amejiaq**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/graphExplore.tsx [hunk 1]
```diff
@@ -97,6 +97,26 @@ class GraphExplore {
...
+
+  getUnmergeOption() {
+    return cy.findByTestId("UnmergeOption");
+  }
+
+  getUnmergeSpinner() {
+    return cy.findByTestId("hc-button-component-spinner");
+  }
+
+  getModal() {
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/support/pages/graphExplore.tsx:112]: Can we rename it to know which modal from Explore we are finding?
Example: getMergeModal or getDocCompareModal, etc

File: marklogic-data-hub-central/ui/src/components/explore/graph-explore-side-panel/graph-explore-side-panel.tsx [hunk 2]
```diff
@@ -139,7 +143,7 @@ const GraphExploreSidePanel: React.FC<Props> = (props) => {
     if (currentTab === DEFAULT_TAB) {
       return (
         <div aria-label="instance-view">
-          <TableView document={details} contentType="json" location={{}} isEntityInstance={true} isSidePanel={true} />
+          <TableView document={details} contentType="json" location={{}} isEntityInstance={true} isSidePanel={true} data={data.current} openUnmergeCompare={props.openUnmergeCompare} loadingCompare={props.loadingCompare} isUnmergeAvailable={props.isUnmergeAvailable}/>
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/components/explore/graph-explore-side-panel/graph-explore-side-panel.tsx:146]: Please don't use a single line to have all the properties. It gets unreadable.
See this page as a guide for best practices: https://wiki.marklogic.com/display/ENGINEERING/Data+Hub+Central+React+Best+Practices

File: marklogic-data-hub-central/ui/src/components/explore/graph-view-explore/graph-view-explore.tsx [hunk 3]
```diff
@@ -248,16 +329,36 @@ const GraphViewExplore: React.FC<Props> = (props) => {
   };
 
   const sidePanel = (<div>
-    <GraphExploreSidePanel onCloseSidePanel={onCloseSidePanel} graphView={graphView} />
+    <GraphExploreSidePanel onCloseSidePanel={onCloseSidePanel} graphView={graphView} openUnmergeCompare={openUnmergeCompare} loadingCompare={loading} data={props.data} isUnmergeAvailable={isUnmergeAvailable}/>
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/components/explore/graph-view-explore/graph-view-explore.tsx:332]: Same here about the length of the line

File: marklogic-data-hub-central/ui/src/components/explore/graph-explore-side-panel/graph-explore-side-panel.tsx (hunk 2 above)
Comment (amejiaq) [reply marklogic-data-hub-central/ui/src/components/explore/graph-explore-side-panel/graph-explore-side-panel.tsx]: Done!, Thanks for the info link!

File: marklogic-data-hub-central/ui/src/components/explore/graph-view-explore/graph-view-explore.tsx (hunk 3 above)
Comment (amejiaq) [reply marklogic-data-hub-central/ui/src/components/explore/graph-view-explore/graph-view-explore.tsx]: Done!

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/graphExplore.tsx (hunk 1 above)
Comment (amejiaq) [reply marklogic-data-hub-central/ui/e2e/cypress/support/pages/graphExplore.tsx]: Done! :)

**Review by MSBrignoni — APPROVED** [review]

**Review by MSBrignoni — APPROVED** [review]

**Review by aniamy — APPROVED** [review]

**Pull request merged**